README.md

```diff
@@ -148,10 +151,14 @@ By default the script runs docker compose to run start the containers and build
 #### Verify contents and set custom env
 The script should have created the supabase folder and also a .env file.
 - Review the env variables and correct or change to suit your environment - the env should already contain sensible defaults (however)
-    - Set in your OPENAI key  [OPENAI_API_KEY] 
+    - Set in your OPENAI key  [`OPENAI_API_KEY`] 
```

Comment:
How does this work for non-OpenAI models?

Reply:
Good spot! :smile:

Pushed a change which stops this being a requirement, which it was until now!